### DISCUSSION TITLE
News feed for new scrips

### DISCUSSION THREAD

**Discussion by DSDV** [tteck/Proxmox#2767] [General] [answered]
Hello, 
sorry if this has come up before. 

I really like the website (https://tteck.github.io/Proxmox/) 
and brows it from time to time for inspiration / new projects 
Lately i noticed that there are even more interesting LXCs here:  https://github.com/tteck/Proxmox/tree/55175f9b315372cf961be494c9125f0bfaacbf0d/install
and i wonder if it maybe is an idea to have some kind of update feed where one can look through all the nice and new scripts ? 

or maybe some tab on the website with a small timeline of newly added scrips<3

with kind regards

**Comment by tteck** [comment, marked as answer]
[CHANGELOG](https://github.com/tteck/Proxmox/blob/main/CHANGELOG.md) not good enough?

**Reply by DSDV** [reply]
uhh thats exactly what i was looking for 😍
if you like i think it would be nice to have it linked on the homepage (:

**Reply by tteck** [reply]
It is! lol

**Reply by DSDV** [reply]
lol i needed to ctrl+f search it XD
my brain completely ignored those buttons o.o
